tests/test_aggregate_verdict.py

```
@@ -1072,3 +1072,223 @@ def test_all_four_verdict_types_in_stats(self, tmp_path):
+            "verdict": "FAIL", "summary": "Critical security issue.",
+            "stats": {"critical": 1, "major": 0, "minor": 0, "info": 0},
+        }))
+        (tmp_path / "apollo.json").write_text(json.dumps({
+            "reviewer": "APOLLO", "perspective": "correctness",
+            "verdict": "PASS", "summary": "ok",
+        }))
+        override = json.dumps({
+            "actor": "pr-author", "sha": "abc1234", "reason": "False positive"
+        })
+        policies = json.dumps({"SENTINEL": "maintainers_only", "APOLLO": "pr_author"})
+        code, out, err = run_aggregate(
```

Ruff (0.14.14)

[warning] Unpacked variable `out` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)